### ISSUE TITLE
[ERROR] False L1, L2, L3 positive with backslash in string

### ISSUE THREAD

**Issue by ManiGillier** [Epitech/coding-style-checker#18]
## Prerequisites

* [x] I check that an issue does not already exists for the error I spotted.
* [x] I can not debug and fix that error by myself through a pull request.

## Error's description

The following code is detected as a false positive : 

```c
"\e[48;5;%dm \e[0m"
```

In itself, it is an error of compilation, but put in a good form (examples below), it is detected as errors of coding-style

### First example
```c
static const char *str = "\e[48;5;%dm \e[0m";
```
Errors : L2, L3
### Second example
```c
/*
** EPITECH PROJECT, 2024
** test
** File description:
** test
*/

#include <stdio.h>

int main(void)
{
    const int color = 250;

    dprintf(2, "\e[48;5;%dm \e[0m", color);
}
```
Errors: L1, L3 (not L2 for some reason)

I think the problem is due to the backslash (\\) in the string that breaks the checker and comes out of the string.

Hopefully you have everything needed to reproduce the issue ! 
If you need anything more to reproduce the error, I will answer in the chat of this issue.

Thank you very much.

**Comment by ManiGillier** [comment]
Oh right thank you i'm gonna correct that mistake !

**Comment by Sigmanificient** [comment]
Will wait to see your issue on the other repo
